unittests/integration/draw_tester.cpp

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+#include "CUDAQTestUtils.h"
+#include <cudaq/algorithms/draw.h>
+
+CUDAQ_TEST(DrawTester, checkEmpty) {
+
+  auto kernel = []() __qpu__ {};
+
+  std::string expected_str = "<empty trace>";
+  auto produced_str = cudaq::draw(kernel);
+  EXPECT_EQ(expected_str, produced_str);
+}
+
+CUDAQ_TEST(DrawTester, checkOps) {
+
+  auto kernel = []() __qpu__ {
+    cudaq::qvector q(3);
+    h(q);
+    x<cudaq::ctrl>(q[0], q[1]);
+    z(q[2]);
+    r1(3.14159, q[0]);
```

Comment:
What does the plot look like when
a) the rotation angle is a variable
b) the precision of the variable is higher/full double precision?